security/v1/consts.go

```diff
@@ -10,4 +10,7 @@ const (
 	// This annotation pins required SCCs for core OpenShift workloads to prevent preemption of custom SCCs.
 	// It is being used in the SCC admission plugin.
 	RequiredSCCAnnotation = "openshift.io/required-scc"
+
+	// MinimallySufficientPodSecurityStandard indicates the PodSecurityStandard that matched the SCCs available to the users of the namespace.
+	MinimallySufficientPodSecurityStandard = "security.openshift.io/psa"
```

Comment:
```suggestion
	MinimallySufficientPodSecurityStandard = "security.openshift.io/MinimallySufficientPodSecurityStandard"
```

Reply:
Maybe a nit: overall we seem to be using both kebab-case and camelCase for annotations/labels, so not sure which one is preferable -- but probably PascalCase should be avoided?

i.e. suggesting
```
security.openshift.io/minimallySufficientPodSecurityStandard
```
or
```
security.openshift.io/minimally-sufficient-pod-security-standard
```
instead of
```
security.openshift.io/MinimallySufficientPodSecurityStandard
```